docs/dxp/latest/en/content-authoring-and-management/content-dashboard/content-dashboard-interface.md

```diff
@@ -23,7 +23,7 @@ To access the Content Info sidebar, click the *Actions Menu* (![Action Menu](../
 
 ## Content Audit Tool
 
-This tool is the graphical area at the top of the Content Dashboard page. The chart represents the total number of assets for the combination of categories in the global Vocabularies you select. This chart takes into account:
+This tool is the graphical area at the top of the Content Dashboard page. The chart represents the total number of assets for the combination of vocabularies categories including *global* and all *sites & asset libraries* in the instance. This chart takes into account:
```

Comment:
It doesn't show the combination of all vocabularies categories, just the ones that you select.